src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -102,6 +102,29 @@ public static TypedData ToRpc(this object value)
                     }
                 }
 
+                // parse ClaimsPrincipal if exists
+                if (request.HttpContext?.User?.Identities != null)
+                {
+                    foreach (var id in request.HttpContext.User.Identities)
+                    {
+                        // asdf
+                        var rpcId = new RpcClaimsIdentity();
+                        if (id.AuthenticationType != null)
+                        {
+                            rpcId.AuthenticationType = id.AuthenticationType;
+                        }
```

Comment:
grpc throws if you any of the values are explicitly set to null. Add null checks to all the values

Reply:
I wasn't aware of this behavior. Thank you; adding null checks!